internal/infrastructure/db/models/integration_features.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+type IntegrationFeatures struct {
+	FeatureKey      string `json:"feature_key,omitempty"`
+	DisplayName     string `json:"display_name,omitempty"`
+	IconCSS         string `json:"css_icon,omitempty"`
+	ElasticProperty string `json:"elastic_property,omitempty"`
+}
+
+type DeviceIntegrationFeatures struct {
```

Comment:
I think we probably only want SQLBoiler models in this folder—typically we run `sqlboiler psql --no-tests --wipe`, which deletes the whole `models` folder.